backend-services/voting-app/src/main/java/org/cardano/foundation/voting/service/auth/web3/KeriWeb3Filter.java

```diff
@@ -125,53 +126,40 @@ protected void doFilterInternal(HttpServletRequest req,
 
         // Step 1: Check if OOBI is already registered
         Either<Problem, String> oobiCheckResult = keriVerificationClient.getOOBI(headerOobi, 1);
```

Comment:
Not super familiar with ballot but shouldn't the OOBI always be verified? We only need to pass the OOBI once during registration/user verification

Reply:
From inspection that seems to only be done for discord verification. It would make more sense if its done for both there and just never check it here. That way you don't need to pass the OOBI here either, just signature, payload and identifier.

But it's late in the game so OK to just leave it

Reply:
You are right. The registration and checking of the OOBI should be done only when verifying the wallet and not every time an auth request is received